spring-xd-yarn/site/config/servers.yml

```diff
@@ -88,8 +88,8 @@ spring:
 #      mapreduce.application.classpath: '$PWD/mr-framework/hadoop/share/hadoop/mapreduce/*:$PWD/mr-framework/hadoop/share/hadoop/mapreduce/lib/*:$PWD/mr-framework/hadoop/share/hadoop/common/*:$PWD/mr-framework/hadoop/share/hadoop/common/lib/*:$PWD/mr-framework/hadoop/share/hadoop/yarn/*:$PWD/mr-framework/hadoop/share/hadoop/yarn/lib/*:$PWD/mr-framework/hadoop/share/hadoop/hdfs/*:$PWD/mr-framework/hadoop/share/hadoop/hdfs/lib/*:/usr/phd/3.0.1.0-1/hadoop/lib/hadoop-lzo-0.6.0.3.0.1.0-1.jar:/etc/hadoop/conf/secure'
 ## For hdp23 only (values for version 2.3.0.0, also change resourceManagerPort above to 8050)
 #    config:
-#      mapreduce.application.framework.path: '/hdp/apps/2.3.0.0-249/mapreduce/mapreduce.tar.gz#mr-framework'
-#      mapreduce.application.classpath: '$PWD/mr-framework/hadoop/share/hadoop/mapreduce/*:$PWD/mr-framework/hadoop/share/hadoop/mapreduce/lib/*:$PWD/mr-framework/hadoop/share/hadoop/common/*:$PWD/mr-framework/hadoop/share/hadoop/common/lib/*:$PWD/mr-framework/hadoop/share/hadoop/yarn/*:$PWD/mr-framework/hadoop/share/hadoop/yarn/lib/*:$PWD/mr-framework/hadoop/share/hadoop/hdfs/*:$PWD/mr-framework/hadoop/share/hadoop/hdfs/lib/*:/usr/hdp/2.3.0.0-249/hadoop/lib/hadoop-lzo-0.6.0.2.3.0.0-249.jar:/etc/hadoop/conf/secure'
+#      mapreduce.application.framework.path: '/hdp/apps/2.3.0.0-2557/mapreduce/mapreduce.tar.gz#mr-framework'
```

Comment:
My mistake in a previous commit - just changed the version part but forgot the build no.